apps/passport-server/src/routing/routes/genericIssuanceRoutes.ts

```diff
@@ -78,13 +76,55 @@ export function initGenericIssuanceRoutes(
     }
   );
 
-  // temporary -- just for testing JWT authentication
   app.get(
-    "/generic-issuance/api/user/ping",
+    "/generic-issuance/api/pipelines",
     async (req: express.Request, res: express.Response) => {
       checkGenericIssuanceServiceStarted(genericIssuanceService);
-      await genericIssuanceService.authenticateStytchSession(req);
-      res.json("pong");
+      const { id } =
+        await genericIssuanceService.authenticateStytchSession(req);
+      res.send(await genericIssuanceService.getAllUserPipelineDefinitions(id));
+    }
+  );
+
+  app.get(
+    "/generic-issuance/api/pipelines/:id",
+    async (req: express.Request, res: express.Response) => {
+      checkGenericIssuanceServiceStarted(genericIssuanceService);
+      const { id: userId } =
+        await genericIssuanceService.authenticateStytchSession(req);
```

Comment:
i suggest adding a function like `authenticateSession` that uses stytch in prod and authenticates in a different way in testing.

Reply:
https://github.com/proofcarryingdata/zupass/issues/1460